docx/header.py

```diff
@@ -34,6 +35,13 @@ def is_linked_to_previous(self):
             return True
         return False
 
+    @lazyproperty
+    def body(self):
+        """
+        BlockItemContainer instance with contents of Header
+        """
+        return BlockItemContainer(None, self)
```

Comment:
Eventually this needs to be instantiated with the `<w:hdr>` element from `/word1/header1.xml` (or whatever), not `None`.

I know we need to add this to `docx/oxml/__init__.py`:
```
from .header import CT_Hdr
register_element_cls('w:hdr',      CT_Hdr)
```
But not quite sure how we want to knit together all the xml files, the XmlParts, and the elements here...